maps/randomvaults/objects.dm

```diff
@@ -174,7 +177,7 @@
 
 /obj/machinery/power/apc/no_alerts/vault_listening/initialize()
 	. = ..()
-	name = "\improper Listening Outpost APC."
+	name = "\improper Listening Outpost APC"
```

Comment:
It's a bad habit to get into. You will see a lot of mistakes as you go through the code, but unless its relevant to what you're doing, you should leave them alone.